pkg/bindings/util/util.go

```diff
@@ -1,19 +1,24 @@
 package util
 
 import (
+	"errors"
+	"net/url"
 	"reflect"
 	"strconv"
+	"strings"
+
+	jsoniter "github.com/json-iterator/go"
 )
 
-func IsSimpleType(f reflect.Value) bool {
+func isSimpleType(f reflect.Value) bool {
```

Comment:
This is technically an API break.  While I doubt anybody is using it, I would sleep better keeping the two functions exported.

Retrospectively, it may should have been an `pkg/bindings/*internal*/util` where we can change whenever we want to.

Reply:
I introduced that just 10 ago so it's probably not used by anybody. I really should have added to some package with internal in its name.

Reply:
Oh sorry, I should have checked if it's in v3.0!

In this case, could you move this code into `pkg/bindings/internal/util`?  This way, it's not part of the public API and we can change as much as we want without having to worry about potential external users.

Reply:
Note that the code can be moved in another PR.  It's (almost) green, so we can merge this one.

Reply:
It was merged 10 days ago to master, and then backported to v3. So it was released 3 days ago with v3.